### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/DrizzlePac/align_mosaics/align_mosaics.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#395]
## What
An accessibility scan flagged 30 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/DrizzlePac/align_mosaics/align_mosaics.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<table class="table">` (selector: `.table`)
- `<pre id="codecell1">` (selector: `#codecell1`)
- `<pre id="codecell2">` (selector: `#codecell2`)
- `<pre id="codecell6">` (selector: `#codecell6`)
- `<pre id="codecell10">` (selector: `#codecell10`)
- `<table id="table140705547325520" class="table-striped table-bordered table-condensed">` (selector: `#table140705547325520`)
- `<pre id="codecell11">` (selector: `#codecell11`)
- `<table id="table140705562311376" class="table-striped table-bordered table-condensed">` (selector: `#table140705562311376`)
- `<pre id="codecell12">` (selector: `#codecell12`)
- `<table id="table140706963273616" class="table-striped table-bordered table-condensed">` (selector: `#inspect-the-wcs-solution > .cell.container.docutils:nth-child(5) > .cell_output.container.docutils > .text_html.output > div > .table-striped.table-bordered.table-condensed`)
- `<pre id="codecell13">` (selector: `#codecell13`)
- `<table id="table140705547962960" class="table-striped table-bordered table-condensed">` (selector: `.cell.container.docutils:nth-child(7) > .cell_output.container.docutils > .text_html.output > div > .table-striped.table-bordered.table-condensed`)
- `<pre id="codecell15">` (selector: `#codecell15`)
- `<pre id="codecell16"><span></span>2025296183816 INFO src=drizzlepac.haputils.astrometric_utils- Created catalog &apos;ref_cat.ecsv&apos; with 1061  sources
</pre>` (selector: `#codecell16`)
- `<pre id="codecell17">` (selector: `#codecell17`)
- `<pre id="codecell18">` (selector: `#codecell18`)
- `<pre id="codecell19">` (selector: `#codecell19`)
- `<table id="table140706963273616" class="table-striped table-bordered table-condensed">` (selector: `#inspect-the-shift-file-to-verify-the-pointing-residuals > .cell.container.docutils:nth-child(6) > .cell_output.container.docutils > .text_html.output > div > .table-striped.table-bordered.table-condensed`)
- `<pre id="codecell23">` (selector: `#codecell23`)
- `<pre id="codecell25">` (selector: `#codecell25`)
- `<pre id="codecell27">` (selector: `#codecell27`)
- `<pre id="codecell28">` (selector: `#codecell28`)
- `<pre id="codecell33">` (selector: `#codecell33`)
- `<pre id="codecell34">` (selector: `#codecell34`)
- `<pre id="codecell35">` (selector: `#codecell35`)
- `<table id="table140705547962960" class="table-striped table-bordered table-condensed">` (selector: `#id1 > .cell.container.docutils:nth-child(6) > .cell_output.container.docutils > .text_html.output > div > .table-striped.table-bordered.table-condensed`)
- `<pre id="codecell39">` (selector: `#codecell39`)
- `<pre id="codecell42">` (selector: `#codecell42`)
- `<pre id="codecell43">` (selector: `#codecell43`)
- `<pre id="codecell44">` (selector: `#codecell44`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
